File: client/src/app/_models/lineParams.ts
import { Line } from "./line";

export class LineParams {
    currentLineId : number;
    searchBy: string  ='symbol';
    searchListOptions: string[] = ['symbol','id'];
    searchText: string = "";
    orderBy  = 'currentLineId'
    orderByListAsc: String[] = [
        'notificationsAmountAsc',
        'idAsc',
        'symbolAsc',
        'lengthAsc'
    ];
    orderByListDesc: String[] = [
        'notificationsAmountDesc', 
        'idDesc',
        'symbolDesc',
        'lengthDesc'
    ];
    headerTableNames: String [] = [
        'Not. amount',
        'Id',
        'Symbol',
        'Length'
    ];
    pageNumber = 1;
    pageSize: number = 5; 
    
    constructor(){

    }
}

File: API/Helpers/LineParams.cs
namespace API.Helpers
{
    public class LineParams : PaginationParams
    {
        public int CurrentLineId { get; set; }    
        public string SearchText { get; set ;}
        public string SearchBy { get; set; }
        public string OrderBy { get; set; } //= "currentLineId";

    }
}